collect-adb.rst

```diff
@@ -0,0 +1,115 @@
+*****************************************
+Using Android Debug Bridge with Collect
+*****************************************
+
+This document focuses specifically on using **adb** as a command line tool to perform tasks related to Collect. 
+
+.. _what-is-adb:
+
+What is Android Debug Bridge (adb)?
+====================================
+
+Android Debug Bridge or `adb <https://developer.android.com/studio/command-line/adb.html>`_ is a command line tool which acts as a bridge between the Android device and the terminal. It can control device over USB from a computer, copy files back and forth, install and uninstall apps, run shell commands, and more. To install **adb**, please follow the instructions given `here <https://android.gadgethacks.com/how-to/android-basics-install-adb-fastboot-mac-linux-windows-0164225/>`_. For the developers and users of ODK Collect, the most common uses are:
+
+- include pushing blank forms to SD Card
```

Comment:
delete "include"

Reply:
I think "loading blank forms" is more consistent with our docs.

 - The task is to "load forms"
 - To do that, you `adb push`